### ISSUE TITLE
Updating from 2.0.1 to 3.0.0 breaks the tests

### ISSUE THREAD

**Issue by andreborud** [jhildenbiddle/karma-file-fixtures-preprocessor#3]
Just updated your plugin from 2.0.1 and karma from 4.2.0 to 4.4.0. Doing that `window.__FIXTURES__` is suddenly undefined.

I have config file similar to this:

```
/*global module */
process.env.CHROME_BIN = require('puppeteer').executablePath()
const isDocker = require('is-docker')()

module.exports = function (config) {
	'use strict'

	config.set({
		// Use cwd as base path
		basePath: '',

		// Set the browser to run
		browsers: ['ChromeHeadlessNoSandbox'],
		concurrency: Infinity,

		plugins: [
			...
			'karma-file-fixtures-preprocessor',
		],

		...

		preprocessors: {
			...
			'test/resources/**/*.json'   : ['file-fixtures']
		}
	})
}
```

Keeping karma at 4.4.0 and downgrading to 2.0.1 make it work again. 
Is there an update to what the configuration should look like?

**Comment by p0lar-bear** [comment]
[Here](https://github.com/jhildenbiddle/karma-file-fixtures-preprocessor/commit/d7eb3126ddd290190835b1aca1605da7692668fc#diff-168726dbe96b3ce427e7fedce31bb0bcR49) lies the issue: it looks like a check was added to make sure that fixture file match patterns weren't also in the list of karma's file match patterns. However, it isn't doing the check properly; the `isInFilePatterns` is simply the result of `CONFIG_FILE_PATTERNS.indexOf(key)`, when it should be `CONFIG_FILE_PATTERNS.indexOf(key) !== 1` like the `isFileFixturePath` flag right before it.

Downgrading for now seems to be the workaround until this is fixed.

**Comment by jhildenbiddle** [comment]
Apologies for the slow response, @andreborud and @p0lar-bear. I recently relocated which has been keeping me busy.

Thanks for tracking down the issue, @p0lar-bear. Very much appreciated. I've made the fix and published a new version of the plugin (3.0.1).
